http/auth.go

```diff
@@ -15,9 +16,16 @@ import (
 	verifier "github.com/okta/okta-jwt-verifier-golang"
 )
 
+type key string
+
 const (
-	sessionName       = "okta-session"
-	sessionIDTokenKey = "id_token"
+	// IDTokenKey is the key used for id_token in session store and context injection
+	IDTokenKey key = "id_token"
+
+	sessionName                  = "okta-session"
+	sessionIDTokenKey     string = string(IDTokenKey)
```

Comment:
If we were passing anything other than a plain string to the session store, we would have to register our custom type with the cookie store for serialization. Just casting to a string is simpler.